bindings/matrix-sdk-ffi/src/platform.rs

```diff
@@ -291,18 +211,12 @@ pub fn setup_otlp_tracing(config: OtlpTracingConfiguration) {
     #[cfg(target_os = "android")]
     log_panics();
 
-    let otlp_tracer =
```

Comment:
What is the point of this method if it doesn't do anymore what it's advertised to do? The `OtlpTracingConfiguration` also seems to be pointless?

Reply:
Indeed, you're right!